src/stream-analytics/azext_stream_analytics/_help.py

```diff
@@ -0,0 +1,370 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=line-too-long
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['stream-analytics job'] = """
+    type: group
+    short-summary: Commands to manage stream-analytics streaming job.
+"""
+
+helps['stream-analytics job create'] = """
+    type: command
+    short-summary: Creates a streaming job or replaces an already existing streaming job.
+    examples:
+      - name: Create a streaming job
+        text: |-
+               az stream-analytics job create --resource-group "sjrg6936" --name "sj59" \\
+               --location "West US" --events-out-of-order-policy "Drop" \\
+               --output-error-policy "Drop" --events-out-of-order-max-delay-in-seconds "5" \\
+               --events-late-arrival-max-delay-in-seconds "16" --data-locale "en-US"
+"""
+
+helps['stream-analytics job update'] = """
+    type: command
+    short-summary: Updates existing streaming job.
+    examples:
+      - name: Update a streaming job
+        text: |-
+               az stream-analytics job update --resource-group "sjrg6936" --name "sj59" \\
+               --events-out-of-order-max-delay-in-seconds "21" \\
+               --events-late-arrival-max-delay-in-seconds "13"
+"""
+
+helps['stream-analytics job delete'] = """
+    type: command
+    short-summary: Deletes a streaming job.
+    examples:
+      - name: Delete a streaming job
+        text: |-
+               az stream-analytics job delete --resource-group "sjrg6936" --name "sj59"
+"""
+
+helps['stream-analytics job show'] = """
+    type: command
+    short-summary: Gets details about the specified streaming job.
+    examples:
+      - name: Get a streaming job
+        text: |-
+               az stream-analytics streaming-job show --resource-group "sjrg6936" --name "sj59"
+      - name: Get a streaming job and expand its inputs, outputs, transformation, and functions
+        text: |-
+               az stream-analytics streaming-job show --resource-group "sjrg3276" --name "sj7804" --expand
+"""
+
+helps['stream-analytics job list'] = """
```

Comment:
is there list by subs?

Reply:
nice finding, list by subscription added.